css/style.css

```diff
@@ -1,22 +1,34 @@
 /* global */
 @import url("https://fonts.googleapis.com/css?family=Roboto");
 
+html {
+  height: 100%;
+}
+
 body {
-  /*
-  margin: 0;
-  padding: 0; */
+  display: table;
+  position: absolute;
+  height: 100%;
+  width: 100%;
   font-family: "Roboto", sans-serif;
   background-color: #101214;
   color: #7a7c80;
 }
 
 .jumbotron {
+  height: 100%;
+  display: table-cell;
+  vertical-align: middle;
   background-color: #101214 !important;
 }
 
 h2,
 .white {
-  color: rgb(10, 110, 87);
+  color: #92f6de;
+}
+
+h2 {
+  user-select: none;
```

Comment:
didn't want anyone to select my name by accident when going through the main entrance sequence.